blockchain/blockchain-wallet-v4-frontend
# Fix: signup clean

## Description 
1. Added checkmark svg to alerts with `success` status.
2. Added opacity on-hover highlighting to "Don't have a wallet?".

## Testing Steps (optional)
1. Go to signup, mouse over "Don't have a wallet?". 
2. Submit send, notice alert popup with checkmark.
